src/pint/models/stand_alone_psr_binaries/binary_generic.py

```diff
@@ -219,7 +219,7 @@ def add_binary_params(self, parameter, defaultValue, unit=False):
                 if unit:
                     log.warning(
                         "Binary parameters' value generally has unit."
-                        " Treat parameter " + parameter + " as a diemension less unit."
+                        " Treat parameter " + parameter + " as a dimension less unit."
```

Comment:
Is there a reason not to make this an f-string?  Also "dimensionless"

Reply:
done